src/sass/_Fabric.Color.Variables.scss

```diff
@@ -21,19 +21,20 @@ $ms-color-themeLighterAlt: #eff6fc;
 
 //== Grayscale Colors
 //
-$ms-color-black:               #000000;
-$ms-color-neutralDark:         #212121;
-$ms-color-neutralPrimary:      #333333;
-$ms-color-neutralPrimaryAlt:   #3C3C3C;
-$ms-color-neutralSecondary:    #666666;
-$ms-color-neutralSecondaryAlt: #767676;
-$ms-color-neutralTertiary:     #a6a6a6;
-$ms-color-neutralTertiaryAlt:  #c8c8c8;
-$ms-color-neutralLight:        #eaeaea;
-$ms-color-neutralLighter:      #f4f4f4;
-$ms-color-neutralLighterAlt:   #f8f8f8;
-$ms-color-white:               #ffffff;
-
+$ms-color-black:                 #000000;
+$ms-color-neutralDark:           #212121;
+$ms-color-neutralPrimary:        #333333;
+$ms-color-neutralPrimaryAlt:     #3C3C3C;
+$ms-color-neutralSecondary:      #666666;
+$ms-color-neutralSecondaryAlt:   #767676;
+$ms-color-neutralTertiary:       #a6a6a6;
+$ms-color-neutralTertiaryAlt:    #c8c8c8;
+$ms-color-neutralQuarternaryAlt: #d0d0d0;
+$ms-color-neutralQuarternary:    #dadada;
```

Comment:
Spelling: Should be neutralQuaternary :)

Also, Quaternary/QuaternaryAlt should be flipped--typically the dark colors are declared before their lighter "alt" versions.